swap/htlc.go

```diff
@@ -283,10 +281,20 @@ func (h *Htlc) AddSuccessToEstimator(estimator *input.TxWeightEstimator) error {
 			trHtlc.InternalPubKey, successLeaf, timeoutLeafHash[:],
 		)
 
-		estimator.AddTapscriptInput(maxSuccessWitnessSize, tapscript)
+		// The leaf witness size must be calculated without the byte
```

Comment:
IIUC, sizes of elements also consume space in encoded transaction, even though they are not themselves pushed to stack. In sweepbatcher, I added comparison of real tx weight and the estimated weight, logging a warning in case of a discrepancy. The current value of `leafWitnessSize` (with elements' sizes) results in weight estimates equal to real  tx size.